SMARTROBOTICS/AutonomousMobileRobot/CommandReturnMessage/schema.json

```diff
@@ -1,46 +1,40 @@
 {
   "$schema": "http://json-schema.org/schema#",
-  "$id": "https://smart-data-models.github.io/incubated/SMARTROBOTICS/AutonomousMobileRobot/Command/ReturnMessage/schema.json",
   "$schemaVersion": "0.0.1",
-   "description": "Command return message",
+  "$id": "https://smart-data-models.github.io/dataModel.AutonomousMobileRobot/CommandReturnMessage/schema.json",
+  "description": "Command return message",
   "modelTags": "",
   "type": "object",
-  "allOf": [
-    {
-      "commandTime":{
-      "$ref": "https://smart-data-models.github.io/incubated/SMARTROBOTICS/AutonomousMobileRobot/AutonomousMobileRobot-schema.json#/definitions/commandTime"}
+  "properties": {
+    "type": {
+      "type": "string",
+      "description": "Property. NGSI Entity type. It has to be CommandMessageReturn",
+      "enum": [
+        "CommandMessageReturn"
+      ]
     },
-    {
-      "receivedTime":{
-      "$ref": "https://smart-data-models.github.io/incubated/SMARTROBOTICS/AutonomousMobileRobot/AutonomousMobileRobot-schema.json#/definitions/receivedTime"}
+    "commandTime":{
+      "$ref": "https://smart-data-models.github.io/dataModel.AutonomousMobileRobot/AutonomousMobileRobot-schema.json#/definitions/commandTime"
     },
```

Comment:
for documentation reasons this structure, although it is correct it is not feasible. I'll fix it for something equivalent but 'documentable'